Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/DriveBase.java

```diff
@@ -70,36 +78,46 @@ public DriveBase() {
 
 		leftFrontMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
 		rightFrontMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
-		// leftBackMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
-		// rightBackMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
+
+		// TODO: Look at the manual for "Follower" to see if we need to set the open
+		// loop ramp on the followers.
+		leftBackMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
+		rightBackMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
 	}
 
 	public void initDefaultCommand() {
 		setDefaultCommand(new JoystickDrive());
 	}
 
 	public void drive(Joystick j) {
-		
+
 		drive(j.getRawAxis(RobotMap.LEFTYAXIS), j.getRawAxis(RobotMap.RIGHTYAXIS));
 
-		
-		// leftStatus = ((TalonSRX)
-		// leftBackMotor).getSensorCollection().getPulseWidthRiseToRiseUs();
-		// rightStatus = ((TalonSRX)
-		// rightBackMotor).getSensorCollection().getPulseWidthRiseToRiseUs();
-		//
-		// if (leftStatus == 0 && rightStatus == 0) {
-		// DriverStation.reportWarning("lb aft Sensor Not Present", false);
-		// } else {
-		// DriverStation.reportWarning("lb aft Sensor Present", false);
-		//
-		// }
 	}
 
 	public void drive(double left, double right) {
 		drive.tankDrive(left, right);
 	}
 
+	public void drive(double left, double right, double arc, double distance) {
+		// TODO: make call with drive based on left, right speeds, arc (heading on
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)